src/main/app/src/app/taken/taken-vrijgeven-dialog/taken-vrijgeven-dialog.component.html

```diff
@@ -0,0 +1,20 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2021 Atos
+  ~ SPDX-License-Identifier: EUPL-1.2+
+  -->
+
+<h1 mat-dialog-title>{{'title.taken.vrijgeven' | translate}}</h1>
+
+<div mat-dialog-content>
+    <p *ngIf="data.length == 1">{{'msg.vrijgeven.taak' | translate }}</p>
+    <p *ngIf="data.length != 1">{{ 'msg.vrijgeven.taken' | translate:{aantal: data.length} }}</p>
+</div>
+<div mat-dialog-actions>
+    <button [disabled]="loading" mat-button color="primary" (click)="vrijgeven()">
+        <mat-icon *ngIf="loading">
+            <mat-spinner diameter="20"></mat-spinner>
+        </mat-icon>
+        {{'actie.vrijgeven' | translate}}
+    </button>
+    <button [disabled]="loading" mat-button (click)="close()">{{'actie.annuleren' | translate}}</button>
```

Comment:
missende id

Reply:
Ik heb deze hernoemd naar "dialogClose_button".
Dit beschrijft wat de button doet, ook volgens het juiste format. Maar het is in de context van Taken.
Eens met deze change?

Reply:
Ja lijkt me een prima oplossing